public/apps/configuration/panels/audit-logging/_index.scss

```diff
@@ -5,7 +5,3 @@
 .form-row {
   max-width: 800px;
 }
-
-pre code {
```

Comment:
Before:

<img width="1761" alt="Screenshot 2023-08-18 at 1 45 40 PM" src="https://github.com/opensearch-project/security-dashboards-plugin/assets/17432265/93456d5f-1b1d-4fd5-92b7-fc5737244005">

After:

<img width="1782" alt="Screenshot 2023-08-18 at 1 45 52 PM" src="https://github.com/opensearch-project/security-dashboards-plugin/assets/17432265/a9fbda20-b8c0-49e3-930b-6f0f2d060920">

This is on Security -> Audit Logs -> Compliance Settings (Configure)